frontend/src/Frontend/UI/Dialogs/TxLogs.hs

```diff
@@ -97,20 +104,28 @@ uiTxLogs fileFFI _onExtClose = do
           td "tx-log-row__request-key" $ elClass "span" "request-key-text" $ text
             $ Pact.hashToText $ Pact.unRequestKey $ Api._commandLog_requestKey cmdLog
 
-  _ <- modalFooter $
-    exportButton txLogger
+  -- Both export wallet and txnlog export use the first 8 characters of the first
+  -- key in the wallet. Given that key is deterministic, could we simply generate it
+  -- rather than either loading keys or requiring them to exist in the wallet first?
```

Comment:
Let's figure this out before merging

Reply:
After discussions with Kadena, key prefix is generated instead of looked up.